src/data/moves_info.h

```diff
@@ -13608,12 +13608,11 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
         .target = MOVE_TARGET_BOTH,
         .priority = 0,
         .category = DAMAGE_CATEGORY_SPECIAL,
-        .argument = STATUS1_SLEEP,
```

Comment:
Does the removal of this argument have any particular impact? Why isn't it just replaced per above?

Reply:
Relic Song doesn't use the argument anymore at all. This was just an unused parameter that I thought I should clean up